package.json

```diff
@@ -24,7 +24,7 @@
   "author": "GitHub",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.6.0",
+    "@actions/core": "^1.9.1",
     "@actions/exec": "^1.0.4",
     "@actions/github": "^1.1.0",
     "@actions/http-client": "^1.0.8",
```

Comment:
Update to new version of `http-client`?  https://github.com/actions/toolkit/blob/64c334f0e54c45f0ee319b5707281d9df0c43a3d/packages/http-client/package.json#L3